test/food.spec.js

```diff
@@ -21,13 +21,24 @@ describe('api', () => {
   });
 
   describe('Test food path', () => {
-    test('It can get a single food from the database', async () => {
-       await agent.get('/api/v1/foods/1').then(response => {
+    test('It can get a single food from the database', () => {
+      return request(app).get('/api/v1/foods/1').then(response => {
```

Comment:
I'll update my tests to use this same style, one day we'll figure out this odd async stuff!